assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -1,7 +1,6 @@
 import { jest, describe, test, expect, beforeEach } from "@jest/globals"
-import { render, screen } from "@testing-library/react"
+import { fireEvent, render, screen } from "@testing-library/react"
```

Comment:
Without this, almost all tests in this file fail with this error: 
```
 Invalid LatLng object: (NaN, NaN)
```

Maybe this change should be moved to a PR that comes before this one?

Reply:
Why these tests suddenly start failing with that error as the result of a tooltip... I have absolutely no idea.

Reply:
Yeah, this is also stumping me and I can't find anything relevant in the Testing Library docs. I don't think it's strictly a problem, but still weird.

Reply:
Yeah, I only "figured it out" by asking Kayla about it last week, with the hope (which turned out to be correct!) that this was an error that the team had seen before.